canonical/canonical.com
# WD-15274 - company/about new page

## Done

Add new /company/about page

## QA

- [demo link](https://canonical-com-1407.demos.haus/company/about)
- [copy doc](https://docs.google.com/document/d/19TIwT-GU5wADQmohicj14MnpHEqNqp0422WLunFsbJU/edit?tab=t.0)
- [figma](https://www.figma.com/design/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=13345-17677&node-type=frame&t=G2IQlqsgPRctYv39-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is done correctly according to figma and copy doc

## Issue / Card
[WD-15274](https://warthogs.atlassian.net/browse/WD-15274)

[WD-15274]: https://warthogs.atlassian.net/browse/WD-15274?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
